plugin/src/main/webapp/icons/LICENSE.txt

```diff
@@ -169,4 +169,12 @@ https://github.com/junit-team/junit5/issues/2165#issuecomment-578806575
 Original file: https://camo.githubusercontent.com/049cbfc3f4c820fe5aa0b376a703a975ab75181a/68747470733a2f2f6a756e69
 742e6f72672f6a756e6974352f6173736574732f696d672f6a756e6974352d6c6f676f2e706e67
 The 24x24 and 48x48 icons were created from this file. No modifications were made, except
-the scale that was changed.
\ No newline at end of file
+the scale that was changed.
+
+------------------------------------------------------------------------------
+
+Hadolint icons:
+
+Taken from https://hadolint.github.io/hadolint/img/cat_container.png and rescaled.
```

Comment:
Does Hadolint have a mailing list or chat? Can you please ask for a permission to reuse the icon? Icons are sometimes problematic to reuse.

Reply:
Sure I'll do so.

Reply:
I now have feedback at https://github.com/hadolint/hadolint/issues/449#issuecomment-658594755 - is sufficient from your point of view?

Reply:
Yes, that is fine now!